mongo/integration/unified/unified_spec_runner.go

```diff
@@ -27,9 +27,16 @@ var (
 		// GODRIVER-1773: This test runs a "find" with limit=4 and batchSize=3. It expects batchSize values of three for
 		// the "find" and one for the "getMore", but we send three for both.
 		"A successful find event with a getmore and the server kills the cursor (<= 4.4)": "See GODRIVER-1773",
+
 		// TODO(GODRIVER-2843): Fix and unskip these test cases.
 		"Find operation with snapshot":                                      "Test fails frequently. See GODRIVER-2843",
 		"Write commands with snapshot session do not affect snapshot reads": "Test fails frequently. See GODRIVER-2843",
+
+		// TODO(GODRIVER-3043): Avoid Appending Write/Read Concern in Atlas Search
+		// Index Helper Commands.
+		"dropSearchIndex ignores read and write concern":       "Sync GODRIVER-3074, but skip testing bug GODRIVER-3043",
+		"listSearchIndexes ignores read and write concern":     "Sync GODRIVER-3074, but skip testing bug GODRIVER-3043",
+		"updateSearchIndex ignores the read and write concern": "Sync GODRIVER-3074, but skip testing bug GODRIVER-3043",
```

Comment:
The [searchIndexIgnoresReadWriteConcern.json ](https://github.com/mongodb/specifications/blob/master/source/index-management/tests/searchIndexIgnoresReadWriteConcern.json) tests are supposed to be added in GODRIVER-3043. However, this ticket was backlogged. Since some of the tests pass and because doing this ticket first could put GODRIVER-3043 out of sync I decided to include those tests here and ignore the failures.